File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        
        s = s.toLowerCase();
        int count = 0;

        for (int i = 0; i < s.length(); i++) {
            
            if (s.charAt(i) == 'p') {
                count += 1;
            } 
            else if (s.charAt(i) == 'y') {
                count -= 1;
            }
        }
        
        if (count == 0) {
            return true;
        }else {
            return false;
        }
        
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):    
    s = s.lower()
    
    p_count = s.count("p")
    y_count = s.count("y")
    
    if p_count == y_count:
        return True
    if p_count != y_count:
        return False
    if p_count == 0 and y_count == 0:
        return True
